kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -225,14 +272,20 @@
   /**
    * Update a {@link Dataset} for the given dataset or view URI.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * You can add, remove, or change the datatype of columns in your dataset,
```

Comment:
I think ["datatype" should be "data type"](https://books.google.com/ngrams/graph?content=data+type%2C+datatype&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cdata%20type%3B%2Cc0%3B.t1%3B%2Cdatatype%3B%2Cc0)